Dockerfile

```diff
@@ -1,10 +1,10 @@
 FROM swift:4.2.4
 WORKDIR /app
-ADD . ./
+COPY . ./
 RUN git submodule update --init --recursive
 RUN swift package clean
 RUN swift build -c release
 RUN mkdir /app/bin
-RUN mv `swift build -c release --show-bin-path` /app/bin
+RUN mv "swift build -c release --show-bin-path" /app/bin
```

Comment:
@HananMehmood5 I don't think this will work.
The result of `swift build -c release --show-bin-path` needs to be used in the `mv` command. Hence the use of `

With double quotes, I get this:

```shell
=> ERROR [8/8] RUN mv "swift build -c release --show-bin-path" /app/bin                                                                                                                                     0.2s
------
 > [8/8] RUN mv "swift build -c release --show-bin-path" /app/bin:
#12 0.176 mv: cannot stat 'swift build -c release --show-bin-path': No such file or directory
```